hack/test-examples.sh

```diff
@@ -3,6 +3,9 @@ set -eu -o pipefail
 
 ./dist/argo delete -l workflows.argoproj.io/test
 
+# Grant admin privileges for the default service account so we could test the examples that submit k8s resources.
+kubectl create rolebinding default-admin --clusterrole=admin --serviceaccount=argo:default -n argo
```

Comment:
@alexec Is this okay? Or should we keep additional RBAC manifests that's more fine-grained in a separate folder?